packages/OptionButton/src/OptionButton.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import {
+  TextIcon,
+  OptionScaleIcon,
+  MultipleChoiceIcon,
+  LikertIcon,
+  TextIconDisabled,
+  OptionScaleIconDisabled,
+  MultipleChoiceIconDisabled,
+  LikertIconDisabled,
+} from './svgs';
+
+import './option-button.scss';
+
+export type ButtonType = 'Text' | 'OptionScale' | 'MultipleChoice' | 'Likert';
+
+export interface OptionButtonProps extends React.ComponentProps<'input'> {
+  /** True when the option button is selected */
+  checked?: boolean;
+  /** Add a specific class to the option button */
+  className?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Add a description that is displayed below the text prop */
+  description?: string;
+  /** True if the option button should be disabled */
+  disabled?: boolean;
+  /** Add an icon the the beginning of the option button.
+   * This overrides the icon that comes with buttonType */
+  icon?: React.ReactNode;
```

Comment:
Did you validate with Francis this case? Is it possible that this component is a specific icon?

Reply:
yes, he said adding the icon prop was fine.